ui/react-components/components/Conflicts/ServiceUnavailableConflicts.jsx

```diff
@@ -5,16 +5,20 @@ import React from "react";
 import PropTypes from "prop-types";
 import {getAppointmentConflictDetails} from "./ConflictsUtil";
 import {sortBy} from "lodash";
+import { ListItem, UnorderedList } from "carbon-components-react";
 
 const ServiceUnavailableConflicts = props => {
 
     const getConflictsList = () => {
         let conflictList = [];
         const conflicts = sortBy(props.conflicts, conflict => conflict.startDateTime);
         conflicts.forEach((conflict, index) => {
-            conflictList.push(<div className={classNames(appointmentConflict)} key={index}>
-                <div className={classNames(conflictDetails)}>{getAppointmentConflictDetails(conflict)}</div>
-            </div>);
+            conflictList.push(
```

Comment:
pushing to a new list inside a forEach is a complete anti-pattern. That is precisely what the .map() function is for.

const newArray = array.map(x => <div>{x}</div>).

I understand this is existing code, but can we improve this?